src/routes/+layout.svelte

```diff
@@ -26,37 +26,38 @@
         <AppBar>Hazy Goods</AppBar>
     </svelte:fragment>
 
-    <AppRail>
-        <svelte:fragment slot="lead">
-            <AppRailAnchor href="/">
-                <div class="ml-7">
-                    <Icon
-                        src={BuildingStorefront}
-                        size="20px"
-                        theme="outline"
-                    />
-                </div>
-                Home</AppRailAnchor
-            >
-            <AppRailAnchor href="/dispensary">
-                <div class="ml-7">
-                    <Icon
-                        src={ClipboardDocumentList}
-                        size="20px"
-                        theme="outline"
-                    />
-                </div>
-                Dispensary List</AppRailAnchor
-            >
-            <AppRailAnchor href="/dispensary/add">
-                <div class="ml-7">
-                    <Icon src={PlusCircle} size="20px" theme="outline" />
-                </div>
-                Add New Dispensary</AppRailAnchor
-            >
```

Comment:
Tags should not have whitespace before the trailing `>` and ideally the closing tag should line up at the same indentation depth as the opening tag.  See below for the cleaner formatting (imo), it seems that Prettier is not finding these cases by default, so perhaps there is something we can configure to enable this formatting automatically.